src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -299,18 +318,24 @@ export class Artwork extends React.Component<Props, State> {
 
     return (
       <>
-        <FlatList<ArtworkPageSection>
-          keyboardShouldPersistTaps="handled"
-          data={this.sections()}
-          ItemSeparatorComponent={() => (
-            <Box mx={2} my={3}>
-              <Separator />
-            </Box>
-          )}
-          refreshControl={<RefreshControl refreshing={this.state.refreshing} onRefresh={this.onRefresh} />}
-          contentContainerStyle={{ paddingBottom: 40 }}
-          renderItem={({ item }) => (item.excludePadding ? item.element : <Box px={2}>{item.element}</Box>)}
-        />
+        {this.state.fetchingData ? (
+          <Flex style={{ flex: 1, height: "100%" }} flexDirection="row" justifyContent="center" alignItems="center">
+            <Spinner />
+          </Flex>
```

Comment:
```suggestion
          <ProvidePlaceholderContext>
            <AboveTheFoldPlaceholder />
          </ProvidePlaceholderContext>
```

**Suggestion:** Let's use the same placeholder/loading indicator we use when entering the screen.

Reply:
Good idea to use the placeholder! Done